config/datanode.example.toml

```diff
@@ -576,6 +576,39 @@ apply_on_query = "auto"
 ## - `[size]` e.g. `64MB`: fixed memory threshold
 mem_threshold_on_create = "auto"
 
+## The options for bloom filter in Mito engine.
+[region_engine.mito.bloom_filter]
+
+## Whether to create the bloom filter on flush.
+## - `auto`: automatically (default)
+## - `disable`: never
+create_on_flush = "auto"
+
+## Whether to create the bloom filter on compaction.
+## - `auto`: automatically (default)
+## - `disable`: never
+create_on_compaction = "auto"
+
+## Whether to apply the bloom filter on query
+## - `auto`: automatically (default)
+## - `disable`: never
+apply_on_query = "auto"
+
+## Memory threshold for bloom filter creation.
+## - `auto`: automatically determine the threshold based on the system memory size (default)
+## - `unlimited`: no memory limit
+## - `[size]` e.g. `64MB`: fixed memory threshold
+mem_threshold_on_create = "auto"
+
+## Cache size for bloom filter metadata.
+metadata_cache_size = "4MiB"
+
+## Cache size for bloom filter content.
+content_cache_size = "128MiB"
+
+## Page size for bloom filter content cache.
+content_cache_page_size = "8MiB"
```

Comment:
non-blocking: Configuring cache size for each index type is verbose. We may need to provide a global index cache.